app/rsr/controllers/getPredictorScores.js

```diff
@@ -1,17 +1,25 @@
 const { Controller } = require('hmpo-form-wizard')
+const { getDraftPredictorScore } = require('../../../common/data/hmppsAssessmentApi')
+const logger = require('../../../common/logging/logger')
+const { splitPredictorScores } = require('../../../common/utils/predictors')
 
 class GetPredictorScores extends Controller {
```

Comment:
what's the difference now between this new predictor controller and the old one where we have removed some code - `app/predictorScores/get.controller.js`, could we unify this in one single controller for predictor?

Reply:
We probably could - I just moved some of the supporting functions to the util file so we could access them from our new flow.

Reply:
i think we should come back and do some good cleanup and add more tests coverage, once we are happy with the new framework and want to decommission the old one on Fe and Be as well.